cluster/engine.go

```diff
@@ -0,0 +1,508 @@
+package cluster
```

Comment:
I believe this should be either `git mv`ed or `git cp`ed as it looks like git thinks it's a brand new file